docs/getting-started/prerequisites.md

```diff
@@ -0,0 +1,237 @@
+# Setting up the prerequisites
+
+This guideline includes the steps to install the prerequisites with specified versions for the Carbyne Stack. All installation procedures are done with the Terminal application of Ubuntu. **Please open the Terminal application before you proceed.**
+
+## Prerequisites
+
+This part of the tutorial is developed and tested with Ubuntu 20.04. Please refer to this [link](https://ubuntu.com/tutorials/install-ubuntu-desktop) for Ubuntu installation steps.
+
+## Platform Setup Prerequisites
+
+Software to be installed:
+
+- [go](https://go.dev/doc/install)
+- [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) v20.10.6
+- [Kind](https://kind.sigs.k8s.io/) v0.11.0
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) v1.21.1
+- [Helm](https://helm.sh/docs/intro/install/) v3.7.1
+
+### go
+
+The go language is a prerequisite for the Docker Engine. In this guideline go 1.18 will be installed.
+
+1. Download go for Linux.
+
+    ```shell
+    wget https://go.dev/dl/go1.18.linux-amd64.tar.gz
+    ```
+
+2. Extract to /usr/local. You may need to insert the password for sudo permissions.
+
+    ```shell
+    sudo tar -C /usr/local -xzf go1.18.linux-amd64.tar.gz
+    ```
+
+3. Check if there is a “go” folder in "/usr/local".
+
+    ```shell
+    ls /usr/local
+    ```
+ 
+4. Update PATH for go.
+
+    ```shell
+    echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc
+    source ~/.bashrc
+    ```
+
+5. Verify if PATH is updated. Now go 1.18 is successfully installed.
+
+    ```shell
+    go version
+    ```
+
+### Docker Engine
+
+1. Update repository index.
+
+    ```shell
+    sudo apt-get update
+    sudo apt-get install ca-certificates curl gnupg lsb-release
+    ```
+
+2. Add Docker’s official GPG key.
+
+    ```shell
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+    ```
+
+3. Set up the stable repository.
+
+    ```shell
+    echo \
+      "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
+      $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    ```
+
+4. Install Docker Engine.
+
+    ```shell
+    sudo apt-get update
+    sudo apt-get install docker-ce=5:20.10.6~3-0~ubuntu-focal docker-ce-cli=5:20.10.6~3-0~ubuntu-focal containerd.io
```

Comment:
is it a good idea to install a specific version of docker engine? I think it would be better to just install the latest version of docker engine available from docker repository, so we participate from package updates e.g. because of some security fixes

Reply:
I agree. Actually, I submitted this pull request because I found that the whole process of starting up the Millionaires Problem example is a bit messy with all these software components with specified versions (Including Docker and Java). I hope that this update could simplify the deployment process for the current working version of the stack as a starting point.

Reply:
I am a fan of reproducible setups. So I would recommend to keep the version. However, you could "externalize" the version string using a shell variable and then reference the variable in your commands. That would allow people to easily specify another version.